_data/tools.yml

```diff
@@ -756,6 +756,24 @@
       url: https://www.magnitude.com/drivers/trino-odbc-jdbc
     - urltext: Starburst ODBC driver
       url: https://docs.starburst.io/data-consumer/clients/odbc.html
+- name: OpenAPI
+  anchor: openapi
+  category: data-source
+  logo: /assets/images/logos/openapi.png
+  logosmall: /assets/images/logos/openapi-small.png
+  description: |
+    The OpenAPI Specification is a specification language for HTTP APIs that
+    provides a standardized means to define your API to others. You can quickly
+    discover how an API works, configure infrastructure, generate client code,
```

Comment:
The point of using this connector is to avoid having to generate a client and use sql instead. I'd remove this phrase